gr-digital/python/digital/qa_crc32_bb.py

```diff
@@ -193,6 +193,35 @@ def test_007_crc_equal (self):
         # Check that the packets after crc_check are the same as input.
         self.assertEqual(data, sink.data()[0])
 
+    def test_002_crc_equal_unpacked (self):
+        """ Test unpacked operation with packed operation
+        """
+        data = (0, 1, 2, 3, 4, 5, 6, 7, 8)
+        src = blocks.vector_source_b(data)
+        unpack1 = blocks.repack_bits_bb(8, 1, self.tsb_key, False, gr.GR_LSB_FIRST)
+        unpack2 = blocks.repack_bits_bb(8, 1, self.tsb_key, False, gr.GR_LSB_FIRST)
+        crc_unpacked = digital.crc32_bb(False, self.tsb_key, False)
+        crc_packed = digital.crc32_bb(False, self.tsb_key, True)
+        sink1 = blocks.tsb_vector_sink_b(tsb_key=self.tsb_key)
+        sink2 = blocks.tsb_vector_sink_b(tsb_key=self.tsb_key)
+
+        self.tb.connect(
+                src,
+                blocks.stream_to_tagged_stream(gr.sizeof_char, 1, len(data), self.tsb_key),
+                crc_packed,
+                unpack1,
+                sink1
+        )
+        self.tb.connect(
+            src,
+            blocks.stream_to_tagged_stream(gr.sizeof_char, 1, len(data), self.tsb_key),
+            unpack2,
+            crc_unpacked,
+            sink2
+        )
```

Comment:
pedantic, but: indents not identical in the two `connect` calls.

Reply:
shouldn't copy stuff from other places with random indentation.